h/static/scripts/directive/sidebar-tutorial.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+// @ngInject
+function controller($http, session) {
+  var vm = this;
+
+  vm.showSidebarTutorial = function () {
+    return session.state.show_sidebar_tutorial;
+  };
+
+  vm.dismiss = function () {
+    $http({method: 'POST', url: '/sidebar_tutorial/dismiss'});
+    session.state.show_sidebar_tutorial = false;
```

Comment:
I suggest that we shouldn't modify the session state directly here but instead provide methods on the session service to do that. That keeps all modification of the session data in one place.

More generally, what we are doing here is effectively an optimistic update of the session state. It seems quite possible that we might want to do that for other things in future. That will be easier if all modifications happen in one place.